sql/moz-fx-data-shared-prod/firefoxdotcom_derived/ga_sessions_v1/bigconfig.yml

```
@@ -0,0 +1,37 @@
+type: BIGCONFIG_FILE
+tag_deployments:
+- collection:
+    name: Operational Checks
+    notification_channels:
+    - slack: '#de-bigeye-triage'
+  deployments:
+  - column_selectors:
+    - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.ga_sessions_v1.submission_date
```

```suggestion
    - name: moz-fx-data-shared-prod.moz-fx-data-shared-prod.firefoxdotcom_derived.ga_sessions_v1.session_date
```

Looking at the bqetl checks config looks like this should be `session_date` and not `submission_date`?